layout/layout.h

```diff
@@ -174,11 +174,12 @@ enum {
 #ifdef CONFIG_X86_64
 	enum { M0_DEFAULT_LAYOUT_ID = 1 };
 #elif defined CONFIG_AARCH64
-	enum { M0_DEFAULT_LAYOUT_ID = 5 }; 
 /**
- * Pages size on aarch64 Centos 7.9 : 64KB
- * which corresponds to layout Id of 5.
+ * Layout id defines the object unit size which cannot be less than the page size:
+ * 1 - 4KB, 2 - 8KB, 3 - 16KB and so on. The page size on RHEL aarch64 platform is
+ * 64KB, hence the default layout id should be 5 (64KB).
```

Comment:
It's more than 80 chars in length. Please use the latest variant from my suggestion:

```
/*
 * Layout id defines the object unit size which cannot be less than
 * the page size: 1 - 4KB, 2 - 8KB, 3 - 16KB and so on. On RHEL aarch64
 * platform the page size is 64KB, hence the default layout id must be 5.
 * /
 ```

Reply:
Did not understand

Reply:
The lines are longer than 80 chars, please fix.

Reply:
fixed

Reply:
is the line length > 80 char or total content length > 80 char, you are concerned about

Reply:
do you mean to break up the contain line < 80 char each

Reply:
According to Motr coding style, the lines' length should not exceed 80 chars. But looks like you've fixed it already.